containers/health-check-promtail-image/Dockerfile

```
@@ -0,0 +1,25 @@
+#!BuildTag: uyuni/health-check-promtail:latest
+
+ARG BASE=registry.suse.com/bci/bci-base:15.6
+
+FROM $BASE
+
+RUN zypper -n ref
+RUN zypper -n install promtail
```

every RUN is a layer in the image. let's reduce them where we can...

```suggestion
RUN zypper --non-interactive ref && zypper --non-interactive up && \
         zypper --gpg-auto-import-keys --non-interactive install --auto-agree-with-licenses --force-resolution \
                      promtail
```